src/test/java/spoon/test/api/APITest.java

```diff
@@ -355,11 +371,13 @@ public void accept(CtVisitor visitor) {
 
 		final List<CtMethod<?>> setters = Query.getElements(launcher.getFactory(), new SetterMethodWithoutCollectionsFilter(launcher.getFactory()));
```

Comment:
I suggest to check the size of the setters array. At least with the code like
```java
assertTrue(setters.size()>0)
```
actually there is 69 methods and at the time of the bug the collection size was 0.